eventtracking/backends/event_bus.py

```diff
@@ -0,0 +1,26 @@
+"""Event tracker backend that emits events to the event-bus."""
+from openedx_events.analytics.signals import TRACKING_EVENT_EMITTED
+from openedx_events.analytics.data import TrackingLogData
+
+class EventBusRoutingBackend:
+    """
+    Event tracker backend that emits an Open edX public signal.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Event tracker backend that emits an Open edX public signal.
+        """
+
+    def send(self, event):
+        """Emit the Open edX public signal"""
+        tracking_data = TrackingLogData(
+            name = event.get('name'),
+            timestamp = event.get('timestamp'),
+            data = event.get('data'),
+            context = event.get('context')
+        )
+
```

Comment:
You're missing this kind of code annotations here, you can find examples in edx-platform :)